github/pom.xml

```diff
@@ -95,6 +95,27 @@
             <groupId>org.hamcrest</groupId>
             <artifactId>hamcrest-junit</artifactId>
         </dependency>
+
+        <!-- Arquillian recorder reporter -->
+        <dependency>
+            <groupId>org.arquillian.extension</groupId>
+            <artifactId>arquillian-recorder-reporter-spi</artifactId>
+            <version>${version.arquillian.recorder}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.arquillian.extension</groupId>
+            <artifactId>arquillian-recorder-reporter-api</artifactId>
+            <version>${version.arquillian.recorder}</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <!--AssertJ for assertion -->
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>${version.assertj}</version>
```

Comment:
I believe we can clean up dependencies newly added in the modules.

  * move artifacts with version to parent pom `dependencyManagement` section.
  * in all modules refer w/o version.
  * use correct scope - for `assertj` it should be `test`